store/sqlstore/user_store.go

```diff
@@ -576,127 +559,106 @@ func (us SqlUserStore) GetAllProfilesInChannel(channelId string, allowFromCache
 			}
 		}
 
-		var users []*model.User
-
-		query := "SELECT Users.* FROM Users, ChannelMembers WHERE ChannelMembers.ChannelId = :ChannelId AND Users.Id = ChannelMembers.UserId AND Users.DeleteAt = 0"
+		query := usersQuery.
+			Join("ChannelMembers cm ON ( cm.UserId = u.Id )").
+			Where("cm.ChannelId = :ChannelId").Bind("ChannelId", channelId).
+			Where("u.DeleteAt = 0").
+			OrderBy("u.Username ASC")
```

Comment:
Adding an explicit ordering here to simplify unit testing and remove non-determinism, as above.